docs/source/multidc/multidc.md

```diff
@@ -561,3 +561,40 @@ UN  172.16.39.209  336 KB     256          ?       7c30ea55-7a4f-4d93-86f7-c8817
 UN  172.16.25.18   759 KB     256          ?       665dde7e-e420-4db3-8c54-ca71efd39b2e  a
 UN  172.16.87.27   503 KB     256          ?       c19c89cb-e24c-4062-9df4-2aa90ab29a99  c
 ```
+
+## Scylla Manager
+
+Integration with Scylla Manager requires having it installed in just one DC.
+
+In this example, we will choose DC1 Kubernetes Cluster to host it.
+Follow [Deploying Scylla Manager](../manager.md) guide to install it.
+
+In order to define Manager tasks, add them to the ScyllaCluster object deployed in the same Kubernetes cluster where
+your Scylla Manager is running.
+
+Each Scylla Operator creates default Scylla Manager Agent auth token when ScyllaCluster is created, 
+because each Scylla Operator generated different token, we need to make sure that entire cluster use the same auth token.
```

Comment:
```suggestion
because each Scylla Operator instance generates a unique token, we need to make sure that the entire cluster uses the same auth token.
```

Reply:
done